massa-consensus/src/settings.rs

```diff
@@ -200,7 +204,7 @@ impl ConsensusSettings {
     }
 }
 
-/// Consensus configuration
+/// Consensus full configuration (static + user defined)
 /// Assumes thread_count >= 1, t0_millis >= 1, t0_millis % thread_count == 0
```

Comment:
Maybe we can assert that when creating the config ?

Reply:
How do you want to manage to do that? In a test at runtime or statically at compile-time?

Reply:
I don't know what's better ?

Reply:
Compile-time for sure, but I don't know at this time a non-hacky Rust way for doing it ...